ChameleonMiniGUI/FrmMain.cs

```diff
@@ -1329,7 +1329,7 @@ private void OpenChameleonSerialPort()
                     if (!string.IsNullOrEmpty(_firmwareVersion) && _firmwareVersion.Contains("Chameleon"))
                     {
                         _cmdExtension = string.Empty;
-                        txt_output.Text = $"[+] Success, found Chameleon Mini device on '{comPortStr}' with {_deviceIdentification} installed{Environment.NewLine}";
+                        txt_output.Text += $"[+] Success, found Chameleon Mini device on '{comPortStr}' with {_deviceIdentification} installed{Environment.NewLine}";

```

Comment:
We used = here in order to have an empty output box when freshly connected device.   ie, don't want to see old garbage outpu.